README.md

```diff
@@ -1,70 +1,212 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] Thank you for informing us about this in your PR:

![image](https://user-images.githubusercontent.com/57421823/216647921-ff4aee0e-1f97-4612-8212-369cf32b56f0.png)

Now, these are important instructions to run and test your app, kindly please add them to your readme file.  Several people will come to your repo and try to test the app including other reviewers, the final evaluator, and even potential employers.  We want them to easily get to all the project functionality.  Alternatively, you could add an admin user to your `seeds.rb`, make your users run `rails db:seed`, and include the user/password on your readme.   Or, include the seeds file, the instructions to run it, and share the login credentials for your admin user on your review request form. 👍

Reply:
Thank you so much!